### PR TITLE
Remove aprilfools

### PR DESCRIPTION
**Pull request by pingiun**
Merge this after april fools has happened and everyone laughed a lot at the prank

### CODE REVIEW

**Comment by JobDoesburg** [comment]
maybe we want to keep the context preprocessor?

**Comment by pingiun** [comment]
> maybe we want to keep the context preprocessor?

good idea!

File: website/thaliawebsite/settings.py [hunk 1]
```diff
@@ -472,7 +472,6 @@ def from_env(
                 "django.contrib.messages.context_processors.messages",
                 "announcements.context_processors.announcements",
                 "thaliawebsite.context_processors.thumbnail_sizes",
-                "thaliawebsite.context_processors.aprilfools",
```
Comment (JobDoesburg) [inline website/thaliawebsite/settings.py:475]: I think it would also be fine to keep this (but also fine to remove it, so I don't care too much actually)

Comment (pingiun) [reply website/thaliawebsite/settings.py]: Lets also keep this, it makes it less confusing to eventually use again

**Review by JobDoesburg — APPROVED** [review]

**Pull request merged**